uint8 SUCCESS = 1
uint8 FAILED = 0

std_msgs/Header header
uint8 load_state
uint8 mission_id
